docs/spec/operators/socketenricher.md

```diff
@@ -18,4 +18,13 @@ networking ones. The following information can be gotten from a socket object:
 
 ## Parameters
 
-None
+### `socket-enricher-fields`
+
+List of optional fields and their sizes to be enabled on the socket enricher
+using the field0=size,field1=size,... format. Disabling or reducing the size of
+the optional fields can reduce the memory and CPU usage of Inspektor Gadget.
+Passing 0 as the size will use the default size of 512 bytes.
```

Comment:
> Disabling

How can an user disable it? I guess with an empty string, but that should be more explicit.